src/api/ctid-ng.js

```diff
@@ -115,4 +115,12 @@ export default (client: ApiRequester, baseUrl: string) => ({
 
     return client.post(`${baseUrl}/users/me/faxes?${params}`, fax, headers);
   },
+
+  mute: (callId: string) => client.put(`${baseUrl}/users/me/calls/${callId}/mute/start`, null)
+    .then(() => true)
```

Comment:
Can you use `ApiRequester.successResponseParser` as last argument ?

Reply:
Sorry, that was more debugging.